c2corg_ui/templates/outing/view.html

```diff
@@ -163,7 +163,8 @@ <h3 class="heading"><span translate>personal comments</span></h3>
   <button class="btn btn-default scroll-top-btn" onclick="window.scrollTo(0, 0);">&nbsp;<span class="glyphicon glyphicon-menu-up"></span></button>
 </div>
 
-<div class="ng-hide">
+<div class="ng-hide" ng-init="associations = ${outing.get('associations')}">
+  {{detailsCtrl.documentService.associations= associations}}
```

Comment:
how do you want to do it better? it this case, we have to go and pass it through the template because we only receive data with mako. I could write something like `detailsCtrl.setDocumentInDocumentService(document)` but it will still do the same :P